package.json

```diff
@@ -13,7 +13,7 @@
   },
   "scripts": {
     "test": "jest",
-    "publish-coverage": "jest --coverage && cat ./coverage/lcov.info | coveralls",
+    "test:coverage": "jest --coverage",
```

Comment:
tbh, i'd just scrap this and add coverage by default to the test command, and just use that